### PR TITLE
[FIX] Highlight active button in sidebar

### LINKED ISSUE #8500: [Bug] Active link not highlighted in sidebar
In the My account settings view (https://demo.rocket.chat/account/profile). The currently active link in the sidebar is not highlighted. This is disturbing the UX of the app. It would be nice if the currently active window is also highlighted just like it is done in the chat window sidebar. I would love to take on this issue

### PR DESCRIPTION
**Pull request by supra08**
Closes #8500 
This PR adds highlights to the sidebar items in the My Accounts page to make it easy to identify which section is open.

bcyyd-qw887

@ggazzo @tassoevan please review.

### CODE REVIEW

**Comment by tassoevan** [comment]
Is there a way to replace the padding **around** the list with a padding **inside** the items so that the active item looks like when it is hovered?

**Comment by supra08** [comment]
Yep, I have to somehow refer to the parent element of the `<a>`, i.e. the `<li>`, on which the css is applied.

**Comment by tassoevan** [comment]
@supra08 Maybe `const element = e.currentTarget. parentElement;` will do the trick.

**Comment by supra08** [comment]
@tassoevan, I've modified it. Please have a look.
rocket_pr_1

File: packages/rocketchat-ui-sidenav/client/sidebarItem.js [hunk 1]
```diff
@@ -95,7 +95,17 @@ Template.sidebarItem.onCreated(function() {
 });
 
 Template.sidebarItem.events({
-	'click [data-id], click .sidebar-item__link'() {
+	'click [data-id], click .sidebar-item__link'(e) {
+		const element = e.currentTarget;
+		const sidebarElements = document.getElementsByClassName('sidebar-item');
+		for (let i = sidebarElements.length - 5; i < sidebarElements.length; i++) {
```
Comment (tassoevan) [inline packages/rocketchat-ui-sidenav/client/sidebarItem.js]: We shouldn't rely on magic constants in expressions like `sidebarElements.length - 5`. What about using a more scoped query selector get the items?

```
const sidebarElements = document.querySelectorAll('.sidebar .flex-nav .sidebar-item');
for (const sidebarElement of Array.from(sidebarElements)) {
  ...
}
```

Comment (supra08) [reply packages/rocketchat-ui-sidenav/client/sidebarItem.js]: Alright, I'll make the changes and push asap.

**Comment by supra08** [comment]
@tassoevan can you please help me with this? After merging the latest develop, a test is failing. Prior to this the code was working smoothly and the PR is ready.

**Comment by kevinfaguiar** [comment]
Any news about this pull?

**Comment by ashwaniYDV** [comment]
@tassoevan I dont think there is a need to loop around the flex tabs to do the highlight. The highlight feature is already defined in "sidebarItem" Template. 
Please review this PR https://github.com/RocketChat/Rocket.Chat/pull/16300

**Comment by tassoevan** [comment]
I'm closing this PR in favor of #17047. Many things have changed on how we manage the routes deprecating the approach used here. 😞

**Pull request closed without merging**